tools/api-markdown-documenter/src/api-item-transforms/configuration/DocumentationSuiteOptions.ts

```diff
@@ -208,6 +208,11 @@ export interface DocumentationSuiteOptions {
 	 * @remarks Note: this is arbitrary text, and will not be escaped.
 	 *
 	 * @defaultValue No front matter is generated.
+	 *
+	 * @deprecated This feature was never properly defined or designed.
```

Comment:
I'll remove this in a subsequent PR. I don't really see a reason to support this at this level. If consumers want to modify output domain trees with content containing domain-specific syntax, they can modify the output ASTs before rendering. 

FF.com's use case is simply to stick some JSON / HTML-comments at the top of each document. This can be done just as easily at a later stage in the pipeline.